config/m4/ib.m4

```diff
@@ -61,6 +61,8 @@ AS_IF([test "x$with_ib" == xyes],
         save_LDFLAGS="$LDFLAGS"
         save_CFLAGS="$CFLAGS"
         save_CPPFLAGS="$CPPFLAGS"
+        CPPFLAGS="-I$with_verbs/include $CPPFLAGS"
+        LDFLAGS="-L$with_verbs/lib64 $LDFLAGS"
```

Comment:
This is tricky one - yes it is architecture specific, but on some systems it is poorly configured, so linker is looking at lib instead of lib64. I think this was particularly true for some versions of SuSE